File: 001/a.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function main() {
  rl.question("", function(N) {
    const skewers = [];

    rl.question("", function(skewerInput) {
      const skewerValues = skewerInput.split(" ").map(Number);

      for (let i = 0; i < 2 * N; i++) {
        skewers.push(skewerValues[i]);
      }

      skewers.sort((a, b) => a - b);

      let maxIngredients = 0;
      for (let i = 0; i < 2 * N; i += 2) {
        const espetox = skewers[i];
        const espetoy = skewers[i + 1];
        maxIngredients += (espetox < espetoy) ? espetox : espetoy;
      }

      console.log(maxIngredients);

      rl.close();
    });
  });
}

main();

File: 001/a.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace::std;

int main() {
    int N;
    cin >> N;

    vector<int> skewers(2 * N);
    for (int i = 0; i < 2 * N; i++) {
        cin >> skewers[i];
    }

    sort(skewers.begin(), skewers.end());

    int maxIngredients = 0;
    for (int i = 0; i < 2 * N; i+=2) {
        int espetox = skewers[i];
        int espetoy = skewers[i+1];
        maxIngredients += (espetox < espetoy) ? espetox : espetoy;
    }

    cout << maxIngredients << endl;

    return 0;
}
